include/eradius_lib.hrl

```diff
@@ -9,23 +9,7 @@
 -type atom_address() :: {atom_name(), {atom_ip(), atom_port()}}.
 -type atom_address_pair() :: {atom_address(), atom_address()}.
 
-%%- cmds
--define(RAccess_Request,       1).
```

Comment:
No, eradius_lib.hrl is part of the library API, removing definitions from it is a NO-GO.